src/locale/locales/it/messages.po

```diff
@@ -7504,11 +7504,11 @@ msgstr ""
 
 #: src/view/com/util/AccountDropdownBtn.tsx:55
 msgid "This will remove @{0} from the quick access list."
-msgstr ""
+msgstr "Questo rimuoverà @{0} dalla lista d'accesso rapido."
 
 #: src/view/com/util/forms/PostDropdownBtn.tsx:689
 msgid "This will remove your post from this quote post for all users, and replace it with a placeholder."
-msgstr ""
+msgstr "Questo rimuoberà il tuo post da questa citazione per tutti gli utenti, e la rimpiazzerà con un placeholder."
```

Comment:
```suggestion
msgstr "Questo rimuoverà il tuo post da questa citazione per tutti gli utenti, e la rimpiazzerà con un placeholder."
```
I think this might be a typo?